simulator/src/modules/DigitalLed.js

```diff
@@ -16,7 +16,7 @@ import { changeInputSize } from "../modules";
 import { colors } from "../themer/themer";
 
 export default class DigitalLed extends CircuitElement {
-    constructor(x, y, scope = globalScope, color = "Red") {
+    constructor(x, y, scope = globalScope, color = "#ff0000") {
```

Comment:
`Red` is a valid color code btw, you can keep that as well.

Reply:
Greetings Bhaiya, but there is one issue with the `Red` color , as when we keep write `Red` instead of `#ff0000` then
`<input type= 'color'>` shows color black in it's color icon  instead of Red. 
This feature is helpful because not all colors on the color  palette can be easily described using common color names like red, blue, or purple.

let me show this by screenShot
<img width="1680" alt="Screenshot 2024-03-24 at 12 15 55 PM" src="https://github.com/CircuitVerse/CircuitVerse/assets/96644946/aa8d002e-2d5b-4156-ac90-c5ebe2aa6120">

Reply:
then leave it